Assets/Scripts/Animation/AnimationPlayer.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimationPlayer: MonoBehaviour
+{
+    [SerializeField] private Animator playerAnimator;
+
+    [SerializeField] private string animationName;
+
+    [SerializeField] private bool play;

```

Comment:
With this, we have to have one instance of the script for setting the animation parameter to true, and another one for setting it to false. I feel this isn't ideal. 

Perhaps we could have a custom event, something like:
```public class AnimationEvent : UnityEvent<string, bool> { }```

Then we should be able to assign ```Animator.SetBool``` in the inspector. Or use a custom method that wraps it.

When picking up the battery, we could do ```animationEvent.Invoke("ParameterName", true)``` 
When dropping, ```animationEvent.Invoke("ParameterName", false)```

Then we have one event that can handle both turning it on and off. And it is generic so it will work with any parameter key value.

Haven't tested it but something like this should work.